src/globals.test.js

```diff
@@ -3,6 +3,7 @@ import * as globals from './globals';
 import {
   DEFAULT_MAX_SIZE_FOR_REQUEST,
   DEFAULT_MAX_SIZE_FOR_REQUEST_ON_ERROR,
+  DEFAULT_MAX_SIZE_FOR_SPANS_SIZE_STORED_IN_MEMORY,
```

Comment:
This represents the limit on a single span, no? so it should be `SPAN` and not `SPANS`.
Also, I think we should remove the redundant `SIZE` here and name this `DEFAULT_MAX_SIZE_FOR_SPAN_STORED_IN_MEMORY`

Reply:
This is the size limit for all the spans stored in memory. if we are already storing 10 MB of spans in memory and we want to add another then the user will be shown a message and the spans will be dropped.

Fixing the duplicate `SIZE` in the name